wgpu/src/lib.rs

```diff
@@ -4739,7 +4765,42 @@ impl<'a> Drop for RenderPass<'a> {
     }
 }
 
-impl ComputePass {
+impl<'a> ComputePass<'a> {
+    /// Drops lifetime constraint to the parent command encoder, making usage of the encoder
+    /// while this pass is recorded a runtime error instead.
+    ///
+    /// Attention: As long as the compute pass has not been ended, any mutable operation on the parent
+    /// command encoder will cause a runtime error and invalidate it!
+    /// By default the lifetime constraint protects from this, but it can be useful
+    /// to handle this at runtime, for example in order to store the pass and encoder in the same
```

Comment:
```suggestion
    /// to handle this at run time, for example in order to store the pass and encoder in the same
```

(At least in the version of English in my head, it's properly spelled “at run time” like “at compile time”, and “runtime” is only the right spelling for “the runtime” in the sense of helper code.)